examples/dialogflow-entities-example/main.py

```diff
@@ -0,0 +1,226 @@
+
+    entities = [{
+        "display_name":
+            "saving-account-types",
+        "kind":
+            dialogflow_v2.entity_types_client.enums.EntityType.Kind.KIND_MAP,
+        "entities": [{
+            "value":
+                "saving-account-types",
+            "synonyms": [
+                "saving", "saving account", "child saving", "IRA", "CD",
+                "student saving"
+            ]
+        }]
+    }, {
+        "display_name":
+            "checking-account-types",
+        "kind":
+            dialogflow_v2.entity_types_client.enums.EntityType.Kind.KIND_MAP,
+        "entities": [{
+            "value":
+                "checking-account-types",
+            "synonyms": [
+                "checking", "checking account", "student checking account",
+                "student account", "business checking account",
+                "business account"
+            ]
+        }]
+    }, {
+        "display_name":
+            "account_types",
+        "kind":
+            dialogflow_v2.entity_types_client.enums.EntityType.Kind.KIND_LIST,
+        "entities": [{
+            "value": "@saving-account-types:saving-account-types",
+            "synonyms": ["@saving-account-types:saving-account-types"]
+        }, {
+            "value": "@checking-account-types:checking-account-types",
+            "synonyms": ["@checking-account-types:checking-account-types"]
+        }, {
+            "value":
+                "@sys.date-period:date-period "
+                "@saving-account-types:saving-account-types",
+            "synonyms": [
+                "@sys.date-period:date-period "
+                "@saving-account-types:saving-account-types"
+            ]
+        }, {
+            "value":
+                "@sys.date-period:date-period "
+                "@checking-account-types:checking-account-types",
+            "synonyms": [
+                "@sys.date-period:date-period "
+                "@checking-account-types:checking-account-types"
+            ]
+        }]
+    }]
+
+    # Example of a EntityTypeBatch dictionary used for entity type batch.
+    # More info:
+    #   https://github.com/googleapis/googleapis/blob/551cf1e6e3addcc63740427c4f9b40dedd3dac27/google/cloud/dialogflow/v2/entity_type.proto#L533
+    # For each entity type in the batch:
+    #   If `name` is specified, we update an existing entity type.
+    #   If `name` is not specified, we create a new entity type.
+    #   The name is the the unique identifier of the entity type.
+    # More info:
+    #   https://github.com/googleapis/googleapis/blob/master/google/cloud/dialogflow/v2/entity_type.proto#L397
+    #   https://dialogflow-python-client-v2.readthedocs.io/en/latest/_modules/dialogflow_v2/gapic/entity_types_client.html#EntityTypesClient.batch_update_entity_types
+
+    entities_batch = {
+        "entity_types": [{
+            "name":
+                "projects/<project_id>/agent/entityTypes/51637fc0-f717-4458-aef1-3d6a51d113f5",
```

Comment:
Seems like we're hard coding too much.
What is the significance of this uuid?
can project_id be inferred or parameterized?